### ISSUE TITLE
[$10]Submission page skipped heading level 

### ISSUE THREAD

**Issue by veshu** [topcoder-platform/community-app#3185]
1 Go to challenge submission page
2. After h1 we have h4 instead of h2

![image](https://user-images.githubusercontent.com/2202045/62267780-1dfb4a00-b44d-11e9-9eb0-42524afb7518.png)

Related issue https://github.com/topcoder-platform/community-app/issues/2799

**Comment by veshu** [comment]
@topcoder-platform/topcodercompetitors new easy fix issue open for pickup. Thanks!

**Comment by crazyk07** [comment]
[500]: Challenge for the updated issue 3185 is creating, rescheduling this event<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by crazyk07** [comment]
[500]: Challenge for the updated issue 3185 is creating, rescheduling this event<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by crazyk07** [comment]
[500]: Challenge for the updated issue 3185 is creating, rescheduling this event<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by crazyk07** [comment]
[504]: Failed to create challenge.<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by crazyk07** [comment]
[504]: Failed to create challenge.<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by crazyk07** [comment]
Contest https://www.topcoder.com/challenges/30098297 has been created for this ticket.<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by crazyk07** [comment]
Contest https://www.topcoder.com/challenges/30098297 has been updated - it has been assigned to anonymousjaggu.<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by anonymousjaggu** [comment]
fixed in https://github.com/topcoder-platform/community-app/pull/3186

**Comment by crazyk07** [comment]
Payment task has been updated: https://software.topcoder.com/review/actions/ViewProjectDetails?pid=30098297<br/><br/>```This is an automated message for crazyk via Topcoder X```
